common/utils.h

```diff
@@ -20,6 +20,10 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
+#ifdef __cplusplus
```

Comment:
Yes, see https://github.com/DynamoRIO/drmemory/wiki/Code-Reviews: "The final squash-and-merge step will squash these additional commits with the original to make a single commit that will be fast-forward-merged into master."  Please read and follow the squash-and-merge commit editing instructions at https://github.com/DynamoRIO/drmemory/wiki/Code-Reviews#merging-into-master